totalRP3/Modules/Register/Characters/RegisterUICharacteristics.xml

```diff
@@ -365,28 +375,92 @@ https://raw.githubusercontent.com/Meorawr/wow-ui-schema/main/UI.xsd">
 					<Anchor point="LEFT" x="15"/>
 				</Anchors>
 			</Frame>
-			<Frame parentKey="Name" inherits="TRP3_TruncatedTextTemplate">
-				<Size x="135"/>
-				<KeyValues>
-					<KeyValue key="fontStringTemplate" value="GameFontNormal"/>
-					<KeyValue key="fontStringJustifyH" value="LEFT"/>
-				</KeyValues>
+		</Frames>
+		<Layers>
+			<Layer level="OVERLAY">
+				<FontString parentKey="Title" font="GameFontNormal" inherits="GameFontNormal" mixin="TRP3_TruncatedTextMixin" justifyH="LEFT">
```

Comment:
is it normal that font and inherits are both there with the same font referenced? Seeing this in a few new fontstrings.

Reply:
It was a leftover from when I was deciding how best to deal with the fontstring mixin changes.